src/ameba/rule/style/parentheses_around_condition.cr

```diff
@@ -32,12 +33,14 @@ module Ameba::Rule::Style
       description "Disallows redundant parentheses around control expressions"
 
       exclude_ternary false
+      exclude_multiline false
       allow_safe_assignment false
     end
 
     MSG_REDUNDANT = "Redundant parentheses"
     MSG_MISSING   = "Missing parentheses"
 
+    # ameba:disable Metrics/CyclomaticComplexity
```

Comment:
do we want to break down the method to reduce the complexity? If we don't follow our own suggetions, who would follow? 😄